_rules/form-field-non-empty-accessible-name-e086e5.md

```diff
@@ -22,7 +22,19 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any element that is [included in the accessibility tree](#included-in-the-accessibility-tree), and that has one of the following [semantic roles][]: `checkbox`, `combobox` (`select` elements), `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch`, `textbox`.
+This rule applies to any element that is [included in the accessibility tree](#included-in-the-accessibility-tree), and that has one of the following [semantic roles][]:
```

Comment:
I don't think this should be written out in a list like this. I think we've more recently set a president that in cases where we have a bunch of rules, we put the list in the background, and have the applicability say what the conditions are for the roles. This rule has it currently flipped the wrong way around.